app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009-2014 Martin Szulecki
+		   2008-2010 Jonathan Beck
+		   2009 Nikias Bassen
+		   2008 Matt Colyer
```

Comment:
For COPYRIGHT and LICENSE we don't put the closing double quotes in a separate line but at the last line. And we only put one tab starting at the second line. The result should look like this:
```
COPYRIGHT="2009-2014 Martin Szulecki
	   2008-2010 Jonathan Beck
	   2009 Nikias Bassen
	   2008 Matt Colyer"
```

Yep, these "rules" (about the closing double quotes) are not the same as for {PROVIDES,REQUIRES}{,_devel}...